README.md

```diff
@@ -19,35 +19,38 @@
 
 </div>
 
-# Here is our website to collect data:
-
-[open-assistant.io](https://open-assistant.io)
-
-(project documentation lives [here](https://laion-ai.github.io/Open-Assistant/))
-
 # Table of Contents
 
 - [What is Open Assistant?](#what-is-open-assistant)
-- [Do you want to try it out?](#do-you-want-to-try-it-out)
-- [The Plan](#the-plan)
+- [Useful Links](#useful-links)
+- [How To Try It Out?](#how-to-try-it-out)
 - [The Vision](#the-vision)
-- [How can you help?](#how-can-you-help)
-- [I’m in! How do I contribute?](CONTRIBUTING.md)
+- [The Plan](#the-plan)
+- [How You Can Help](#how-you-can-help)
 
 ---
 
 ## What is Open Assistant?
 
-<p align="center">
```

Comment:
I don't think we want to remove the centering here?

Reply:
I just thought it might be better to remove it for consistency reasons, I'm happy to add that back if you think we should.